jest.config.js

```diff
@@ -2,8 +2,8 @@ module.exports = {
   verbose: true,
   testEnvironment: 'node',
   testMatch: [
-    '**/src/**/*.spec.ts',
-    '**/test/**/*int.spec.ts',
+    '<rootDir>/src/**/*.spec.ts',
+    '<rootDir>/test/**/*int.spec.ts',
```

Comment:
Is anyone opposed to this change? It behaves the same as before, I just wanted to ask as I cannot see why we would need wildcards before the src and test entry points.